### PR TITLE
Initialize components

### PR DESCRIPTION
**Pull request by Bishoy-Samwel**
- [x] Initialise React app.
- [x] Add React Redux (npm install react-redux).
- [x] Structure your application files using a "feature folder" approach and
- [x] use the ducks pattern for your Redux files.
- [x] The building blocks of your app should be set as re-usable components.
- [x] Add React Router and set two <Route>sand <Link>s  for the app's navigation
- [x] Books (Default view - will display the list of books and add/remove book functionality.)
- [x]  Categories (Should display "Under construction" text only.)

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Bishoy-Samwel 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+### `yarn start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### `yarn test`
```
Comment (mikenath223) [inline README.md]: - [x] Kindly update the readme matching [this template](https://github.com/microverseinc/readme-template).

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/jsx-key */
+import React from 'react';
+import Book from './Book';
+
+export default function User() {
```
Comment (mikenath223) [inline src/components/Books.js]: - [x] OPTIONAL - Consider changing this function name to describe the actual functionality of the function.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+// import { store } from './app/store';
+
+ReactDOM.render(
+  <React.StrictMode>
+    {/* <Provider store={store}> */}
+    <Router>
+      <App />
+    </Router>
+    {/* </Provider> */}
+  </React.StrictMode>,
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove the commented code here as comments make the code untidy.

**Comment by mikenath223** [comment]
- [x] Some commit messages aren't descriptive. Please going forward ensure all commit messages actually describe the changes made in the commit.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @Bishoy-Samwel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**